hazelcast/src/test/java/com/hazelcast/internal/networking/nonblocking/iobalancer/IOBalancerMemoryLeakTest.java

```diff
@@ -62,18 +66,23 @@ public void testMemoryLeak_with_RestConnections() throws IOException {
         for (int i = 0; i < 100; i++) {
             communicator.getClusterInfo();
         }
-        final IOBalancer ioBalancer = connectionManager.getIoBalancer();
+        final IOBalancer ioBalancer = getIoBalancer(connectionManager);
         assertTrueEventually(new AssertTask() {
             @Override
             public void run() throws Exception {
                 int inHandlerSize = ioBalancer.getInLoadTracker().getHandlers().size();
                 int outHandlerSize = ioBalancer.getOutLoadTracker().getHandlers().size();
-                Assert.assertEquals(0, inHandlerSize);
-                Assert.assertEquals(0, outHandlerSize);
+                assertEquals(0, inHandlerSize);
+                assertEquals(0, outHandlerSize);
             }
         });
     }
 
+    private static IOBalancer getIoBalancer(TcpIpConnectionManager connectionManager) {
```

Comment:
I think it would match our common pattern to move the static method to the end of the class.